index.bs

```diff
@@ -1477,7 +1486,9 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
         <td>
             Flags (bit 0 is the least significant bit):
             - Bit 0: [=Test of User Presence=] (`TUP`) result.
-            - Bits 1-5: Reserved for future use (`RFU`).
+            - Bit 1: Reserved for future use (`RFU1`).
```

Comment:
In private communication, Alexei Czeskis @leshi let me know that Google is already privately using bit 1 for some purpose, and asked me to use bit 2 for UV.  If Google is willing to move their private use bit elsewhere, I'd be glad to use bit 1 for UV instead, as it would seem more natural.